crates/sui-indexer-alt/src/handlers/wal_coin_balances.rs

```diff
@@ -42,10 +42,10 @@ impl Handler for WalCoinBalances {
                 object_id: value.object_id.to_vec(),
                 object_version: value.object_version as i64,
 
-                owner_id: value.update.as_ref().map(|o| o.owner_id.clone()),
+                owner_id: value.value.as_ref().map(|o| o.owner_id.clone()),
```

Comment:
`value.value` is not super readable. Could we change it to something like `update.new_value` here and in the other `wal-` pipeline?

Reply:
Sure, we can change that.